### DISCUSSION TITLE
Welcome to aolunderground-proggies Discussions!

### DISCUSSION THREAD

**Discussion by ssstonebraker** [ssstonebraker/aolunderground-proggies#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JasonShearer** [comment]
Hey there. I was wondering if anyone happened to have an AOL prog called Chain Reaction. I can't seem to find it anywhere on the internet, including this GitHub collection. There's a couple websites with an old dead link to version 2.6.9 (http://www.freeyellow.com/members2/oxolxloxo/cr269.exe), but that's all I've been able to find, and the file wasn't captured by Internet Archive.
